src/main/scala/devices/debug/Debug.scala

```diff
@@ -383,6 +421,10 @@ class TLDebugModuleOuter(device: Device)(implicit p: Parameters) extends LazyMod
     //  HAMASK is 1 bit per component
     //  HAWINDOWSEL selects a 32-bit slice of HAMASK to be visible for read/write in HAWINDOW
     //--------------------------------------------------------------
+
+    /** hart array mask, will be sent to inner,
+      * doesn't include the hart selected by dmcontrol.hartsello
+      */
```

Comment:
Some duplicated information with the comment block above. Maybe merge two comment blocks?